disnake/client.py

```diff
@@ -2617,7 +2617,7 @@ async def delete_guild_command(self, guild_id: int, command_id: int) -> None:
         command_id: :class:`int`
             The ID of the application command to delete.
         """
-        await self.http.delete_guild_command(self.application_id, guild_id, command_id)
+        return await self._connection.delete_guild_command(guild_id, command_id)
```

Comment:
Why `return`? 🤔

Reply:
For consistency with `delete_global_command`:
https://github.com/DisnakeDev/disnake/blob/4887d39ee95158d28f927aa94b5d3f538b2ba3bd/disnake/client.py#L2478

Removed in https://github.com/DisnakeDev/disnake/pull/907/commits/677dfb12801dc0b291930d469a13dc611d24dccd, since it's really not needed